chain-extensions/xvm/src/lib.rs

```diff
@@ -89,6 +94,20 @@ where
                 // things on behalf of the caller via XVM.
                 let source = env.ext().address();
 
+                // Claim a default account if needed.
+                if value > 0 && UA::to_h160(&source).is_none() {
+                    let claim_result =
+                        pallet_unified_accounts::Pallet::<T>::claim_default_evm_address(
+                            RawOrigin::Signed(source.clone()).into(),
+                        );
+                    if claim_result.is_err() {
+                        return Ok(RetVal::Diverging {
+                            flags: ReturnFlags::REVERT,
+                            data: format!("{:?}", claim_result.err()).into(),
+                        });
+                    }
+                }
+
```

Comment:
Do we need new benchmarks for it? or re-run existing ones for new weights?
I didn't see any todo regarding it.

Reply:
I'll add the weight charging. The weight info from `pallet-unified-accounts` should be enough for it.

Reply:
Updated.